setup.py

```diff
@@ -29,7 +31,8 @@ def get_version():
         'msgpack-python',
         'asgiref>=1.0.0',
     ],
-    extras_requires={
-        "cryptography": ['cryptography>=1.3.0'],
+    extras_require={
+        "cryptography": crypto_requires,
+        "tests": crypto_requires + ['tox', 'asgi_ipc', 'twisted', 'txredisapi'],
```

Comment:
Yes, my idea was that `tests` should contain all dependencies needed to run the tests; it's simpler to the user that way. The tests happen to need `cryptography`, hence the overlap. I think we might need something similar for `daphne` and `channels`, and I'd like to standardize on being able to just install the `tests` extra for any of them to run the tests.

What do you think?